chapters/classes.tex

```diff
@@ -1821,11 +1817,15 @@ \subsubsection{Connections}\label{connections}
 
 The package \lstinline!Connections!\indexinline{Connections} is used for over-constrained connection graphs, \cref{equation-operators-for-overconstrained-connection-based-equation-systems}.
 
-\subsubsection{Graphical Annotation Types}\label{graphical-annotation-types}
+\subsubsection{ExternalObject}\label{externalobject}
 
-A number of ``predefined'' record types and enumeration types for graphical annotations are described in \cref{annotations}.
-These types are not predefined in the usual sense since they cannot be referenced in ordinary Modelica code, only within annotations.
+See \cref{external-objects} for information about the predefined type \lstinline!ExternalObject!.
```

Comment:
I assume the idea was to have the subsubsection with actual contents before the forwarding ones, right?

Reply:
Right.  I don't why the diff ended up looking so incomprehensible.